### ISSUE TITLE
[Enemy Family] Flaming Alts

### ISSUE THREAD

**Issue by dedChar** [ProbablyTez/Agony#486]
Flaming versions of normal enemies like Flaming Hoppers for example. Like with drowned versions of enemies for flooded caves, we should expand on burning enemies for burning basement.

Flaming alts appear in burning basement. Normal enemies can transform into flaming versions by touching fireplaces. Flaming alts are immune to the burn status effect and fireplaces.

**Comment by Anonymous-Koala** [comment]
>Burning Cellar

Ded please.

Should they/or some of them, be able to apply a burning status effect? Also these enemies would mostly be Basement/Cellar enemies.

**Comment by dedChar** [comment]
Oh fuck
It was late when I wrote this
And the backdrop Sprite looks like the cellar Sprite but red anyways

**Comment by dedChar** [comment]
Yeah an occasional burning effect from their attacks would be nice
